apps/site/src/app/global.css

```
@@ -15,314 +12,313 @@
 @source inline("hover:paused");
 
 @theme {
-    --animate-slide-down: slideDown 130s linear infinite;
-    --animate-slide-down-2: slideDown2 130s linear infinite;
-    --animate-slide-up: slideUp 100s linear infinite;
-    --animate-slide-up-2: slideUp2 100s linear infinite;
+  --animate-slide-down: slideDown 130s linear infinite;
```

Stylelint (17.9.0)

[error] Expected "slideDown" to be "slidedown" (value-keyword-case)

(value-keyword-case)